src/mdlComponentHandler.js

```diff
@@ -51,6 +51,24 @@ var componentHandler = (function() {
   }
 
   /**
+   * Searches registered components for matching CSS classes.
+   * @param {Array<string>} classes List of CSS class names we want to find
+   * the matching components of.
+   * @return {Array<object>} The list of corresponding components that have a
+   * matching CSS class. The list can be empty if there is no match.
+   * @private
+   */
+  function findRegisteredComponentsCssClasses_(classes) {
+    var jsClasses = [];
+    for (var i = 0; i < registeredComponents_.length; i++) {
```

Comment:
Or, even better `registeredComponents_.filter()`, although I’m not sure if IE supports that